internal/api/account.go

```diff
@@ -106,6 +106,7 @@ func handleCreateAccount(c *Context, w http.ResponseWriter, r *http.Request) {
 		},
 		AccountMetadata: &model.AccountMetadata{
 			Provision: createAccountRequest.Provision,
+			Subnet:    createAccountRequest.Subnet,
```

Comment:
It seems that when we create account and provide subnet, there is no actual `Subnet` entity assigned to it just the CIDR. I think it may cause some problems down the line, if for example someone will later create  subnet with the same CIDR in database and other account will claim it. It also may cause that for some accounts we could query `SELECT * FROM subnet where AccountID='abcd'` while for others that will not be the case.

Reply:
We now have a check when we claim the subnet to make sure no account is already associated. This way we shouldn't face any issues I think.